generators/client/files-angular.js

```diff
@@ -85,6 +85,14 @@ const files = {
             templates: [
                 { file: 'content/images/hipster.png', method: 'copy' },
                 { file: 'content/images/hipster2x.png', method: 'copy' },
+                { file: 'content/images/hipster-man-1.png', method: 'copy' },
```

Comment:
can we use SVG instead of png? so that we dont have to copy 2 versions, it was done that way in react